src/main/java/core/basesyntax/service/reader/ReaderServiceImpl.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.service.reader;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class ReaderServiceImpl implements ReaderService {
+    @Override
+    public List<String> readFromFile(String inputDataFile) {
+        Path filePath = Paths.get(inputDataFile);
+        try {
+            return Files.lines(filePath)
+                    .skip(1)
```

Comment:
we should be able to read from any file, not only csv, so let's skip the first line when we have all the lines and are already processing them